eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Scripts.hs

```diff
@@ -112,12 +113,12 @@ instance NoThunks Tag
 -- | Scripts in the Alonzo Era, Either a Timelock script or a Plutus script.
 data Script era
   = TimelockScript (Timelock (Crypto era))
-  | PlutusScript ShortByteString -- A Plutus.V1.Ledger.Scripts(Script) that has been 'CBOR' encoded
+  | PlutusScript Language ShortByteString
```

Comment:
Hum. I guess I would have expected either:
```
data Script = TimelockScript ... | PlutusV1Script ... | PlutusV2Script ...
```
or
```
data Script = AScript Language ....
```
or maybe even
```
data Script = TimeLockScript ... | NonNativeScript Language ...
```
IDK, there seems like there's maybe still a bit too much "plutus" specificity (if we're trying to be a bit more agnostic about what languages there are).

Reply:
I was going to cross that bridge when we got there, I don't really know how to anticipate what shape other languages will have.

Reply:
That said, I'm open to suggestions if you have a favorite!